### PR TITLE
flatpak-spawn: Add "--sandbox-a11y-own-name"

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
This is the flatpak-spawn counterpart to the new option of the Flatpak portal.

See https://github.com/flatpak/flatpak/pull/5898

### CODE REVIEW

File: src/flatpak-spawn.c [hunk 1]
```diff
@@ -370,6 +370,28 @@ sandbox_flag_callback (G_GNUC_UNUSED const gchar *option_name,
...
+static gboolean
+sandbox_a11y_own_name_callback (G_GNUC_UNUSED const gchar *option_name,
+                                const gchar *value,
+                                G_GNUC_UNUSED gpointer data,
+                                GError **error)
+{
+  if (sandbox_a11y_own_names == NULL)
+    sandbox_a11y_own_names = g_ptr_array_new ();
+
+  if (!g_dbus_is_name (value) || g_dbus_is_unique_name (value))
```
Comment (TingPing) [inline src/flatpak-spawn.c:384]:
```suggestion
  if (!g_dbus_is_name (value))
```

> Checks if string is a valid D-Bus bus name (either unique or well-known).

**Review by TingPing — APPROVED** [review]
LGTM once the Flatpak portion lands.

File: src/flatpak-spawn.c (hunk 1 above)
Comment (GeorgesStavracas) [reply src/flatpak-spawn.c:384]: My intention with this check was to only allow well-known bus names (and no unique names). Sadly there is no `g_dbus_is_well_known_name()`.

Comment (TingPing) [reply src/flatpak-spawn.c:384]: Ahh, makes more sense. 👍

**Pull request merged**